test/proper_lib.erl

```diff
@@ -39,6 +39,10 @@ date_gen() ->
         proper_types:integer(1, 31)},
        calendar:valid_date(Date)).
 
+utc_offset_gen() ->
+    {proper_types:integer(-15, 15),
+     proper_types:integer(0, 59)}.
```

Comment:
I experimented with this locally. I was able to enter up to `-15:59` and `+15:59` as an offset.